### PR TITLE
Code Quality: Fallback to "C:\" when var doesn't work

### PR DESCRIPTION
**Pull request by yaira2**
**Resolved / Related Issues**

To prevent extra work, all changes to the Files codebase must link to an approved issue marked as `Ready to build`. Please insert the issue number following the hashtag with the issue number that this Pull Request resolves.
- Closes #

**Steps used to test these changes**

Stability is a top priority for Files and all changes are required to go through testing before being merged into the repo. Please include a list of steps that you used to test this PR.

1. Opened Files ...
2. ...

### CODE REVIEW

**Comment by yaira2** [comment]
This didn't solve the issue

File: src/Files.App/Constants.cs [hunk 1]
```diff
@@ -234,6 +234,8 @@ public static class UserEnvironmentPaths
 
 			public static readonly string TempPath = Environment.GetEnvironmentVariable("TEMP") ?? "";
 
+			public static readonly string SystemDrivePath = $@"{Constants.UserEnvironmentPaths.SystemDrivePath}\" ?? @"C:\";
```
Comment (hishitetsu) [inline src/Files.App/Constants.cs:237]: The path parameter should not be included.
```csharp
public static readonly string SystemDrivePath = $@"{Constants.UserEnvironmentPaths.SystemDrivePath}" ?? @"C:";
```

File: src/Files.App/Constants.cs [hunk 2]
```diff
@@ -234,6 +234,8 @@ public static class UserEnvironmentPaths
 
 			public static readonly string TempPath = Environment.GetEnvironmentVariable("TEMP") ?? "";
 
+			public static readonly string SystemDrivePath = Environment.GetEnvironmentVariable("SystemDrive") ?? "C:";
```
Comment (hishitetsu) [inline src/Files.App/Constants.cs:237]: You may want to consider the case where the environment variable is an empty string.

Comment (yaira2) [reply src/Files.App/Constants.cs]: Implemented in [8512bc4](https://github.com/files-community/Files/pull/15842/commits/8512bc47c239528a7bf1d6f59148cafb75fb489e)

File: src/Files.App/Constants.cs [hunk 3]
```diff
@@ -240,6 +240,15 @@ public static class UserEnvironmentPaths
...
 			public static readonly string RecentItemsPath = Environment.GetFolderPath(Environment.SpecialFolder.Recent);
 
+			public static string SystemDrivePath
+			{
+				get
+				{
+					var systemDrive = Environment.GetEnvironmentVariable("SystemDrive");
+					return !string.IsNullOrEmpty(systemDrive) ? systemDrive : "C:";
+				}
+			}
```
Comment (hishitetsu) [inline src/Files.App/Constants.cs:250]:
```suggestion
			public static readonly string SystemDrivePath;

			static UserEnvironmentPaths()
			{
				var systemDrive = Environment.GetEnvironmentVariable("SystemDrive");
				SystemDrivePath = !string.IsNullOrEmpty(systemDrive) ? systemDrive : "C:";
			}
```
The static constructor allows it to remain a readonly field.

**Review by hishitetsu — APPROVED** [review]
LGTM

**Pull request merged**